hrm-service/service-tests/permissions.test.js

```diff
@@ -0,0 +1,68 @@
+const supertest = require('supertest');
+const each = require('jest-each').default;
+const expressApp = require('../src/app');
+import { rulesMap } from '../src/permissions';
+
+const server = expressApp.listen();
+const request = supertest.agent(server);
+
+const headers = {
+  'Content-Type': 'application/json',
+  Authorization: `Basic ${Buffer.from(process.env.API_KEY).toString('base64')}`,
+};
+
+afterAll(done => {
+  server.close(() => {
+    done();
+  });
+});
+
+describe('/permissions route', () => {
+  describe('GET', () => {
```

Comment:
I think checking for the text of the error makes these tests a bit brittle? Perhaps just checking the status code is sufficient